packages/runtime/container-runtime/src/id-compressor/idCompressor.ts

```diff
@@ -7,6 +7,29 @@
 
 import { ITelemetryLogger } from "@fluidframework/common-definitions";
 import { assert } from "@fluidframework/common-utils";
+import {
```

Comment:
The import of legacySharedTreeInitialTreeId in this file (and usages) should be removed (it shouldn't have ended up in the new SharedTree copy anyway).